### ISSUE TITLE
Can we say that FFWD will support Windows Phone Apollo?

### ISSUE THREAD

**Issue by stanleyjoy** [fehaar/FFWD#57]
I am not sure if I should write this here but it is a troubling issue none the less. I was excited when Microsoft was showcasing Windows Phone 8 or Apollo. Game Development on Apollo is very quiet, a little too quite for my comfort.
I was wondering if you guys have heard of any news in Game Development for Apollo and also the main question will Fast Forward support development for Apollo.

Stan

**Comment by fehaar** [comment]
Hi Stan,

As you have probably heard, Microsoft has lifted the veil on WP8.

Game development is given a big lift in the right direction with WP8.
Especially the open support for native C++ and the fact that it will
run with the same core as other Win8 versions is fantastic. It will
mean that native support from engines like Unity and Unreal is
definitely come.

Since XNA will not to be supported for direct development on WP8, it
will be the end of the line for FFWD. The fact that real Unity support
for WP8 will also come at some point, will also make FFWD obsolete. We
had a hunch that something like this would happen when we released the
FFWD source, but we hoped that someone would benefit from it anyway.

Regarding FFWD/XNA development on the 360, nothing has been officially
closed yet. But we can speculate in that it will probably be heading in
the same direction as the phone. But if it will be happening this year
or even in this console generation is anybodys guess.

Sendt fra min Windows Phone
From: Stanley J. Maliackal
Sent: 23-06-2012 10:01
To: Thomas Gravgaard
Subject: [FFWD] Can we say that FFWD will support Windows Phone Apollo?
(#57)
I am not sure if I should write this here but it is a troubling issue
none the less. I was excited when Microsoft was showcasing Windows
Phone 8 or Apollo. Game Development on Apollo is very quiet, a little
too quite for my comfort.
I was wondering if you guys have heard of any news in Game Development
for Apollo and also the main question will Fast Forward support
development for Apollo.

Stan

---

Reply to this email directly or view it on GitHub:
https://github.com/fehaar/FFWD/issues/57

**Comment by stanleyjoy** [comment]
So there is a major probability that Windows Phone Apollo's game development can be done by using a game engine like Unity 3D or Unreal Engine provided Microsoft gives it to them. 
Stan

**Comment by Laumania** [comment]
I really think it's sad that all my hours with XNA is kind of "wasted" if MS will not let XNA be supported on WP8. Win8 will not support XNA by default, which I was hoping for. 
So guess it's time for me to start learning Unity3d, which is actually a good thing :)

**Comment by fehaar** [comment]
Learning a new API and learning to make games is never wasted. Remember that. You will find that you can carry a lot of techniques over to Unity.

I think I have read on some Unity post, maybe just a tweet from David, that they were looking to provide WP8 support, but there is no official word on it yet.

**Comment by stanleyjoy** [comment]
Waiting eagerly on some official news.
@fehaar
Have you got time to check on my issue??

**Comment by Laumania** [comment]
@fehaar Yeah I know and like learning stuff - actually I'm doing it as much as I can. So I had already planned that I would, at some point, look into unity3d because it seems like an awesome game engine.
And you right, a lot of my knowledge about game development isn't really language/platform specific and then talking language I can use C# as scripting for Unity3d, as you know.

Still I find it a little sad if Microsoft kill of there own (pretty ok actually) gamge framework, XNA.

Sorry for going a little off-topic/"spamming" this issue.

**Comment by KonajuGames** [comment]
Both the official Windows Phone Developer Team @wpdevteam on Twitter and Mark Chamberlain in the AppHub forums have stated that XNA will be available in the Window Phone 8 SDK for creating Windows Phone 8 apps.
http://forums.create.msdn.com/forums/p/106054/625197.aspx#625197

**Comment by stanleyjoy** [comment]
Woohoo!!!
FFWD for life, I think!
Stan
@Slygamer
Thanks for the update
@fehaar
What do you think man?? Will we have FFWD for Windows Phone 8??
